modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -114,23 +117,15 @@ const TooltipProvider = () => {
 
 	const saveTitle = (element: HTMLElement) => {
 		if (element) {
-			const title = element.getAttribute('title');
+			const title =
+				element.getAttribute('title') ||
+				element.getAttribute('data-title');
```

Comment:
I don't think we need to check `data-title` and `removeAttribute('data-title')` below. The purpose of this block of code is to remove `title` attribute, so we don't have default browser tooltip in addition to our custom one. `data-title` does not have any effect so it can stick around.

Reply:
I remember in the past that we had uses of data-title as well as title. I could be wrong though, I will look up to see how any uses we have of it.

Reply:
We do use `data-title` in some places, and it should continue to be a trigger for tooltips. But, there is no need to remove and restore it on mouse events. We only need to do that for `title`.

Reply:
@markocikos ah gotcha, that makes sense